services/py-api/src/server/routes/utility_routes.py

```diff
@@ -19,3 +19,9 @@ async def ping(
     response: Response, handler: UtilityHandlers = Depends(create_utility_handler)
 ) -> PongResponse | ErrResponse:
     return await handler.ping_services(response)
+
+@utility_router.get("/random_users", responses={200: {"model": NumberOfRandomParticipantsResponse}, 503: {"model": ErrResponse}})
+async def random_users(
+    response: Response, handler: UtilityHandlers = Depends(create_utility_handler)
+) -> NumberOfRandomParticipantsResponse | ErrResponse:
+    return await handler.random_users_services(response)
```

Comment:
I may understand the purpose of it again, for testing however why would you include it in the commit. Please remove these.

Reply:
Fixed in 1bec98abe406eea6382a5453ac98f3e900d3ddd4